### DISCUSSION TITLE
Selective sync

### DISCUSSION THREAD

**Discussion by ianrego** [TriliumNext/Trilium#4488] [Ideas]
Is there a way to selectively sync notes to a webserver?

I'm looking to start a workgroup and although I love to share my knowledge, there are notes in my trilium that are unnecessary to the workgroup. 

Also, if more than one trilium instance is synching to a webserver installation, will it just merge automatically? In this case I have colleagues who are also using trilium but they have their own instances.

**Comment by zadam** [comment]
What you're asking for is part of a feature set for multi-user cooperation, but that is [explicitly out of scope](https://github.com/zadam/trilium/wiki/FAQ#multi-user-support).

"Selective sync" is not planned.

> Also, if more than one trilium instance is synching to a webserver installation, will it just merge automatically?

Well, currently yes, but in the context of syncing all notes within the document.
